dist/svelboost.es.js

```diff
@@ -0,0 +1,30 @@
,width:!0,height:!0},1);var e=x(r);n=R(e,"circle",{class:!0,cx:!0,cy:!0,r:!0,fill:!0,"stroke-width":!0,"stroke-miterlimit":!0},1),x(n).forEach(v),e.forEach(v),this.h()},h(){_(n,"class","path svelte-r1e5c5"),_(n,"cx",i=t[0]/2),_(n,"cy",o=t[1]/2),_(n,"r",t[2]),_(n,"fill","none"),_(n,"stroke-width",t[3]),_(n,"stroke-miterlimit",t[4]),_(r,"class",s="circular "+t[5]+" svelte-r1e5c5"),_(r,"width",t[0]),_(r,"height",t[1])},m(t,e){g(t,r,e),m(r,n)},p(t,[e]){1&e&&i!==(i=t[0]/2)&&_(n,"cx",i),2&e&&o!==(o=t[1]/2)&&_(n,"cy",o),4&e&&_(n,"r",t[2]),8&e&&_(n,"stroke-width",t[3]),16&e&&_(n,"stroke-miterlimit",t[4]),32&e&&s!==(s="circular "+t[5]+" svelte-r1e5c5")&&_(r,"class",s),1&e&&_(r,"width",t[0]),2&e&&_(r,"height",t[1])},i:e,o:e,d(t){t&&v(r)}}}function Ot(t,e,r){let{width:n=100}=e,{height:i=100}=e,{radius:o=20}=e,{strokeWidth:s=3}=e,{strokeLimit:a="10"}=e,{class:u}=e;return t.$$set=t=>{"width"in t&&r(0,n=t.width),"height"in t&&r(1,i=t.height),"radius"in t&&r(2,o=t.radius),"strokeWidth"in t&&r(3,s=t.strokeWidth),"strokeLimit"in t&&r(4,a=t.strokeLimit),"class"in t&&r(5,u=t.class)},[n,i,o,s,a,u]}class Tt extends X{constructor(t){var e;super(),document.getElementById("svelte-r1e5c5-style")||((e=y("style")).id="svelte-r1e5c5-style",e.textContent="@-webkit-keyframes svelte-r1e5c5-rotate-circle{from{transform:rotate(0deg)}to{transform:rotate(360deg)}}@keyframes svelte-r1e5c5-rotate-circle{from{transform:rotate(0deg)}to{transform:rotate(360deg)}}@-webkit-keyframes svelte-r1e5c5-circular-rotate{0%{transform:rotate(0deg)}100%{transform:rotate(360deg)}}@keyframes svelte-r1e5c5-circular-rotate{0%{transform:rotate(0deg)}100%{transform:rotate(360deg)}}@-webkit-keyframes svelte-r1e5c5-dash{0%{stroke-dasharray:1px, 200px;stroke-dashoffset:0}50%{stroke-dasharray:100px, 200px;stroke-dashoffset:-15px}100%{stroke-dasharray:100px, 200px;stroke-dashoffset:-125px}}@keyframes svelte-r1e5c5-dash{0%{stroke-dasharray:1px, 200px;stroke-dashoffset:0}50%{stroke-dasharray:100px, 200px;stroke-dashoffset:-15px}100%{stroke-dasharray:100px, 200px;stroke-dashoffset:-125px}}.circular.svelte-r1e5c5{position:relative;-webkit-animation:svelte-r1e5c5-circular-rotate 2s linear infinite;animation:svelte-r1e5c5-circular-rotate 2s linear infinite}.path.svelte-r1e5c5{stroke-dasharray:1, 200;stroke-dashoffset:0;-webkit-animation:svelte-r1e5c5-dash 2s ease-in-out infinite;animation:svelte-r1e5c5-dash 2s ease-in-out infinite;stroke-linecap:round;stroke:#0066ff;stroke:var(--primary-accent, #0066ff)}",m(document.head,e)),Q(this,t,Ot,St,s,{width:0,height:1,radius:2,strokeWidth:3,strokeLimit:4,class:5})}}var kt="undefined"!=typeof globalThis?globalThis:"undefined"!=typeof window?window:"undefined"!=typeof global?global:"undefined"!=typeof self?self:{};function $t(t){if(t.__esModule)return t;var e=Object.defineProperty({},"__esModule",{value:!0});return Object.keys(t).forEach((function(r){var n=Object.getOwnPropertyDescriptor(t,r);Object.defineProperty(e,r,n.get?n:{enumerable:!0,get:function(){return t[r]}})})),e}
```

Comment:
@georgeciubotaru 
1. what's the purpose of these files?
2. why do we need them in repo?

Reply:
@zolotokrylin 
1. We are creating reusable components for all types of applications, so users can import it as a module and other methods (https://dev.to/iggredible/what-the-heck-are-cjs-amd-umd-and-esm-ikm).
2. Because we don't publish them to `npm` we need to compile the code and push it into the repo.

Reply:
@georgeciubotaru I read that to import Svelte Components they need not to be pre-built, other way Svelte engine can not do optimizations.

Reply:
![Screenshot_20210707-083521_GitHub.jpg](https://user-images.githubusercontent.com/8217518/124705424-51c73000-defe-11eb-9b29-0a94968d01e8.jpg)